client/components/Cart.js

```diff
@@ -54,8 +77,8 @@ class Cart extends Component {
           $
           { cartTotal }
         </h2>
-        <button>Check Out Now</button>
-        <button>Keep Shopping</button>
+        <button type="button" onClick={(ev) => this.checkout(ev)}>Check Out Now</button>
+        <button type="button" onClick={(ev) => this.keepShopping(ev)}>Keep Shopping</button>
```

Comment:
these can be just {this.checkout}, {this.keepshopping}

Reply:
updated.